app/views/projects/review_turnaround.erb

```diff
@@ -0,0 +1,3 @@
+<div class='p-3'>
+  <%= line_chart  @metrics, ytitle: "Time as minutes", suffix: " minutes" %>
```

Comment:
Can we make it to hours?

Reply:
For example if a review took 30 minutes, the graph should show 0.5 hours.

Reply:
got it! i will review it

Reply:
You didn't change this 🤔

Reply:
sorry, miss click